src/appleseed/renderer/modeling/environmentedf/latlongmapenvironmentedf.cpp

```diff
@@ -595,6 +595,15 @@ DictionaryArray LatLongMapEnvironmentEDFFactory::get_input_metadata() const
             .insert("use", "optional")
             .insert("help", "Environment texture vertical shift in degrees"));
 
+    metadata.push_back(
```

Comment:
I would suggest moving common metadata to a separate function in the base factory class (e.g. `add_common_sky_input_metadata()`), like it's done for some other entity types (sky environment EDFs, and maybe lights, IIRC).